api/receiver/azure_test.go

```diff
@@ -7,67 +7,11 @@
 package receiver
 
 import (
-	"bytes"
-	"io"
-	"os"
-	"path"
-	"runtime"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-// artifact_test.zip has a single dir, artifact_test/, containing 4 files: wpt_report_{1,2}.json, wpt_screenshot_{1,2}.txt.
-
-func readZip(t *testing.T) *os.File {
```

Comment:
I guess these are the tests that need to be rewritten in Python for processor.py?

Reply:
Correct.

Reply:
Done. And even more tests for various downloading functions.